From 9ec2c3fd537f41674d0313ef2c49fb2c204d55cf Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Wed, 29 Jun 2016 18:58:48 -0700
Subject: [PATCH 2/6] fix: Include ncurses tinfo library if found.

Thanks to Fabian Groffen for reporting this issue and providing a
couple possible patches.  From Fabian's report:

  For some time now, ncurses can be built in a mode where the low level
  terminfo functionality lives in a separate lib called libtinfo.
  Because some people do, this means Mutt needs to include this
  library in that case to avoid linking errors [...]
---
 configure.ac | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6096dbb..40e4bbd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -281,7 +281,9 @@ main ()
 	done
         AC_CHECK_LIB($cf_ncurses, initscr,
                 [MUTTLIBS="$MUTTLIBS -l$cf_ncurses"
-		
+
+                AC_CHECK_LIB(tinfo, tgetent, [MUTTLIBS="$MUTTLIBS -ltinfo"])
+
                 if test "$cf_ncurses" = ncursesw; then
 			AC_CHECK_HEADERS(ncursesw/ncurses.h,[cf_cv_ncurses_header="ncursesw/ncurses.h"])
 		else
-- 
2.9.0

